ci/tests/replica.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#imports
+. ci/libs/variables.sh
+. ci/libs/utils.sh
+
+test_single_replica_stop_start() {
+	echo "----------------Test_single_replica_stop_start--------------"
+	orig_controller_id=$(start_controller "$CONTROLLER_IP" "store1" "1")
+	replica1_id=$(start_replica "$CONTROLLER_IP" "$REPLICA_IP1" "vol1")
```

Comment:
```suggestion
	replica1_id=$(start_replica "${CONTROLLER_IP}" "$REPLICA_IP1" "vol1")
```

Reply:
@farhaanbukhsh Why did you suggested this change?